app/src/main/res/layout/recently_played_fragment.xml

```diff
@@ -7,36 +7,13 @@
     android:layout_height="match_parent"
     android:background="@color/color_def_white">
 
-    <com.google.android.material.appbar.AppBarLayout
-      android:id="@+id/reading_list_app_bar_layout"
-      android:layout_width="0dp"
-      android:layout_height="wrap_content"
-      android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent">
-
-      <androidx.appcompat.widget.Toolbar
-        android:id="@+id/recently_played_toolbar"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:background="?attr/colorPrimary"
-        android:fontFamily="sans-serif"
-        android:minHeight="?attr/actionBarSize"
-        app:navigationContentDescription="@string/navigate_up"
-        app:navigationIcon="?attr/homeAsUpIndicator"
-        app:title="@string/recently_played_activity"
-        app:titleTextAppearance="@style/ToolbarTextAppearance"
-        app:titleTextColor="@color/color_def_white" />
-    </com.google.android.material.appbar.AppBarLayout>
-
     <FrameLayout
       android:layout_width="0dp"
       android:layout_height="0dp"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/reading_list_app_bar_layout">
+      app:layout_constraintTop_toTopOf="parent">
```

Comment:
Is the outer ConstraintLayout even needed anymore after this change, or can we move the FrameLayout up to be the root?

Reply:
Removed ConstraintLayout